### ISSUE TITLE
error - perhaps SEV data? 

### ISSUE THREAD

**Issue by RobynIrvine** [poissonconsulting/kootqlt#9]
> dat <- kootqlt:::read_zrxp_data("../../data/zrxp/2016-10")
[1] "../../data/zrxp/2016-10/Poisson_ColumbiaFlocal20161104222833.zrxp"
[1] "../../data/zrxp/2016-10/Poisson_ColumbiaWISKI.zrxp"
Error in `colnames<-`(`*tmp*`, value = c("DateTime", "Level", "Status_BCH" : 
  'names' attribute [3] must be the same length as the vector [1]
Warning message:
../../data/zrxp/2016-10/Poisson_ColumbiaWISKI.zrxp was not inputted 
> dat <- kootqlt:::read_zrxp_data("../../data/zrxp/2016-09")
[1] "../../data/zrxp/2016-09/Poisson_ColumbiaFlocal20161004222829.zrxp"
[1] "../../data/zrxp/2016-09/Poisson_ColumbiaWISKI.zrxp"

**Comment by joethorley** [comment]
The problem is an entry with no data i.e..
```
20161101080000 1156.5
#ZRXPVERSION2209.265|*|ZRXPMODEStandard|*|ZRXPCREATORZRXP-Fileexport|*|TZUTC-8|*|
#REXCHANGEBIR_TW_Raw|*|TSPATH/08NE/202008.1/Source_TW/Cmd.NRTa.O|*|CUNIT?C|*|
##SITENAMECentral Columbia|*|
#RINVAL-777|*|SNAMEColumbia R at Birchbank|*|SANR202008.1|*|CNAMESource_TW|*|
#LAYOUT(timestamp,value)|*|
#ZRXPVERSION2209.265|*|ZRXPMODEStandard|*|ZRXPCREATORZRXP-Fileexport|*|TZUTC-8|*|
#REXCHANGECNN_HG_RawHourMean|*|TSPATH/08NE/202009.1/Source_HG/Hour.Mean.O|*|
#CUNITm|*|RINVAL-777|*|SNAMEColumbia R bl Norns Ck|*|SANR202009.1|*|
##SITENAMECentral Columbia|*|
#CNAMESource_HG|*|
#LAYOUT(timestamp,value)|*|
20161001080000 419.463
```

I'll modify the text so it can deal with these situations.

**Comment by joethorley** [comment]
fixed version 0.1.3
